test/asset_manager_test.rb

```diff
@@ -108,18 +88,12 @@
 
   describe "a Whitehall asset exists" do
     before do
-      stub_asset_manager_has_a_whitehall_asset(
+      stub_asset_manager_has_a_whitehall_media(
```

Comment:
Sorry, all of these will need updating too with the rename

Reply:
of cos the one repo I forgot to check 💯 Thanks for reminding me!